### ISSUE TITLE
Interaction with mathematics without having to install software

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5122]
**Reported by Diego on 2015-05-29 18:05**
Hello developers!
At this time, NVDA need to install the math player 4 to get interact with mathematical content.
It would be possible to implement a form that NVDA does not have to install anything to use this feature?

**Comment by bhavyashah** [comment]
@jcsteh Could you please address this ticket? The requested enhancement does not sound possible to me.

**Comment by ehollig** [comment]
I am leaning towards a cantfix, but am uncertain on NVDA's current implementation with math as I do not use it myself.

**Comment by jcsteh** [comment]
This isn't possible:

1. We can't ship MathPlayer with NVDA, as although it is free, it is still a commercial product and there would be licensing issues. Also, this would significantly increase the size of the NVDA download.
2. At CSUN 2016, I and others did some prototyping of integrating [Speech Rule Engine](https://github.com/zorkow/speech-rule-engine) with NVDA. In future, we might be able to use this instead of MathPlayer. However, because this is written in JavaScript/node.js, we'd have to bundle a node.js runtime, which would unacceptably increase the size of the NVDA download. So, even then, this would be an NVDA add-on.
